templates/customer.json

```diff
@@ -1,18 +1,17 @@
 {
-  "_id": "{{@ system @}}-{{@ datatype @}}-collect",
+  "_id": "{{@ system @}}-customer-collect",
```

Comment:
This won't work, you can't re-use this template in the product and supplier datatypes. I fixed a bug in the connector-cli that did this during collapse, perhaps you are hit by the same.

Reply:
fixed already. Was me.